AirCasting/Utils/DataFormatters.swift

```diff
@@ -0,0 +1,100 @@
+// Created by Lunar on 30/09/2021.
+//
+
+import Foundation
+
+enum DateFormatters {
+    
+    enum SessionDownloadService {
+        static let decoderDataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum SessionUploadService {
+        static let encoderDataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum CreateSessionAPIService {
+        static let encoderDataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum AirBeam3Configurator {
+        static let dataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            df.locale = Locale(identifier: "en_US_POSIX")
+            return df
+        }()
+    }
+    
+    enum SessionCartView {
+        static let pollutionChartDataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "HH:mm"
+            return df
+        }()
+    }
+    
+    enum GraphView {
+        static let dataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "HH:mm"
+            df.locale = Locale(identifier: "en_US")
+            return df
+        }()
+        
+        static let mobileActiveDataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "HH:mm"
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.locale = Locale(identifier: "en_US")
+            return df
+        }()
+    }
+    
+    enum TimeAxisRenderer {
+        static let dataFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeStyle = .short
+            df.locale = Locale(identifier: "en_US")
+            return df
+        }()
+    }
+    
+    enum DateExtension {
+        static let dataFormatter: DateFormatter = {
```

Comment:
typo - its's `date` not `data`. Also - naming is important, please change the name to something more descriptive